runtime/queries/bash/highlights.scm

```diff
@@ -7,45 +7,33 @@
 
 (command_name) @function
 
-(variable_name) @variable.other.member
+(variable_name) @property
```

Comment:
Please revert all the changes in this file because they use incorrect scopes and regress highlighting